include/screen.h

```diff
@@ -5,10 +5,10 @@
 #include "fade.h"
 
 typedef struct {
-    u16 displayControl;
-    u8 filler2[0x2];
-    u16 unk4;
-    u16 displayControlMask;
+    u16 displayControl;     // 0x0
+    u8 filler2[0x2];        // 0x2
+    u16 unk4;               // 0x4
+    u16 displayControlMask; // 0x6
```

Comment:
The usual format for these we have in some places already is to put the offsets on the left, e.g.
```suggestion
    /*0x0*/ u16 displayControl;
    /*0x2*/ u8 filler2[0x2];
    /*0x4*/ u16 unk4;
    /*0x6*/ u16 displayControlMask;
```